e2e/tests/components/column-resizing/column-resizing-sorting/column-resizing-sorting.e2e-spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ColumnResizingSortingPage } from './column-resizing-sorting.po.spec';
+import { imageCompare } from '../../common/image-compare';
+
+
+describe('Column Resizing Sorting Table Tests', () => {
+
+    const page: ColumnResizingSortingPage = new ColumnResizingSortingPage();
+
+    beforeEach(async () => {
+        await page.getPage();
+    });
+
+    it('should not sort when resizing a column', async () => {
+        await page.resizeColumn(0, -100);
+        expect(await imageCompare('column-resize-no-sort')).toEqual(0);
```

Comment:
You should assert the state of the DOM to verify that table rows have not been reordered, this will give better failure information than a screenshot mismatch.